### PR TITLE
feat(secretlint-rule-no-dotenv): add new rule

### LINKED ISSUE #66: rule: .env
`.env` (dot env) is set of credential.
In sometimes, user commit this file by error.

We think about a rule for `.env`.

### PR DESCRIPTION
**Pull request by munierujp**
- Add new rule for [dotenv](https://github.com/motdotla/dotenv)
  - prevent commits `.env`file

close #66

### CODE REVIEW

**Comment by munierujp** [comment]
@azu Could you review this PR?

File: packages/@secretlint/secretlint-rule-no-dotenv/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    ]
+}
+```
+
+## MessageIDs
+
+### FOUND_DOTENV_FILE
+> found .env file
+
+`.env` file was found.
```
Comment (azu) [inline packages/@secretlint/secretlint-rule-no-dotenv/README.md:31]: Can you add Author and LICENSE section?

e.g.) https://github.com/secretlint/secretlint/tree/master/packages/%40secretlint/secretlint-rule-secp256k1-privatekey#author

Comment (azu) [reply packages/@secretlint/secretlint-rule-no-dotenv/README.md:31]: Also add `LICENSE` file.

Secretlint adopt MIT license.

File: packages/@secretlint/secretlint-rule-no-dotenv/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    ],
+    "homepage": "https://github.com/secretlint/secretlint/tree/master/packages/@secretlint/secretlint-rule-no-dotenv/",
+    "bugs": {
+        "url": "https://github.com/secretlint/secretlint/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/secretlint/secretlint.git"
+    },
+    "files": [
```
Comment (azu) [inline packages/@secretlint/secretlint-rule-no-dotenv/package.json:18]: Add author and license https://github.com/secretlint/secretlint/blob/44e1388cbead71f09b2cb81352429002dac53a8c/packages/%40secretlint/secretlint-rule-secp256k1-privatekey/package.json#L23-L24

**Comment by munierujp** [comment]
@azu In this case, who is the "Author"? Creator of tihs package (in other words, me)?  Creator of secretlint (in other words, you)?

**Comment by azu** [comment]
@munierujp Author is cretor of the package. It is you.

File: packages/@secretlint/secretlint-rule-no-dotenv/README.md (hunk 1 above)
Comment (munierujp) [reply packages/@secretlint/secretlint-rule-no-dotenv/README.md:31]: @azu resolved by follows:

- 539f5591ab4d0c6dd7dd532e10a5301e13ae7a21
- 8f2877e0849ebc66eda1efb95d027d2344cd21e8
- cd773aeee5e8685e48bc9a7f72e2810956eb36fa

File: packages/@secretlint/secretlint-rule-no-dotenv/package.json (hunk 2 above)
Comment (munierujp) [reply packages/@secretlint/secretlint-rule-no-dotenv/package.json:23]: @azu resolved by follows:

- 2b19e36ba10ce57034ae8ca939ea61d5efd8ca0a
- 63d9d7bc77c44e1e43c6767b0cd85f8a191c2e5e

**Comment by azu** [comment]
Thanks!

**Pull request merged**